code-challenges.js

```diff
@@ -13,7 +21,19 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+var sortedDataArray1 = fullArr1.filter((Number) => parseInt(Number) === Number);
 
+let newArray = sortedDataArray1.sort((a,b)=> a-b);
+let finalArray = newArray.filter(x => x%2)
+
+console.log(finalArray)
+
+var sortedDataArray2 = fullArr2.filter((Number) => parseInt(Number) === Number);
+
+let newArray2 = sortedDataArray2.sort((a,b)=> a-b);
+let finalArray2 = newArray2.filter(x => x%2)
+
+console.log(finalArray2)
```

Comment:
great, all of the logic is here for this solution! now you just need to put it into a function rather than independent variables/console.logs.